src/query/storages/fuse/src/table_functions/clustering_informations/table_args.rs

```diff
@@ -42,7 +41,7 @@ pub fn parse_func_table_args(table_args: &TableArgs) -> Result<(String, String)>
 pub fn get_cluster_keys(table: &FuseTable, definition: &str) -> Result<Vec<Expression>> {
     let cluster_keys = if !definition.is_empty() {
         let schema = table.schema();
-        let exprs = ExpressionParser::parse_exprs(definition)?;
+        let exprs = unchecked_expressions_analyze(definition)?;
```

Comment:
what if `cluster by (to_date(ts),  -a, b )`

Reply:
`unchecked_expressions_analyze` only supports limited `Expr` types. We can't support complex cluster keys like `(to_date(ts), -a, b)` so far.

`TypeChecker` and `ExpressionBuilder` can parse those expr. But they need `QueryContext` which is not provided in `storage`. We need a refactor here.

Reply:
The test cases of `hits` also failed. Maybe this PR is going the wrong way...